modules/sandboxed-containers-selecting-nodes-web-console.adoc

```diff
@@ -0,0 +1,96 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-selecting-nodes-web-console_{context}"]
+= Installing {sandboxed-containers-first} on selected nodes using the web console
+
+You can use labels to install the Kata runtime on specific worker nodes.
+
+By default, when you create the `KataConfig` custom resource (CR), the {sandboxed-containers-operator} installs the Kata runtime on all worker nodes. Add labels to specific worker nodes and define the label in the `KataConfig` CR when you create it to install the Kata runtime only on the labeled nodes.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
+
+.Procedure
+
+. Label the nodes:
+
+.. From the *Administrator* perspective in the web console, navigate to *Compute* → *Nodes*.
+
+.. Select the node you want to label.
+
+.. In the *Node details* page, click *Actions* and select *Edit labels*.
+
+.. In the *Edit labels* window, enter the new label using this format: `<label_key>=<label_value>`.
+
+.. Click *Save*.
+
+.. Repeat for each node you want to label.
+
+. Create the `KataConfig` CR:
+
+.. Navigate to *Operators* → *Installed Operators*.
+
+.. Select the {sandboxed-containers-operator} from the list of operators.
+
+.. In the *KataConfig* tab, click *Create KataConfig*.
+
+.. In the *Create KataConfig* page, select to configure the `KataConfig` CR via *YAML view*.
+
+.. Copy and paste the following manifest into the YAML view:
++
+[source,yaml]
+----
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+spec:
+  kataConfigPoolSelector:
+    matchLabels:
+    <label_key>: '<label_value>'
+----
+
+.. Click *Create*.
+
+The new KataConfig CR is created and begins to install the kata runtime on the labeled nodes.
+
+[IMPORTANT]
+====
+{sandboxed-containers-first} installs Kata only as a secondary optional runtime on the cluster and not as the primary runtime.
+====
+
+.Verification
+
+. In the *KataConfig* tab, select the new KataConfig CR.
+
+. In the *KataConfig* page, select the *YAML* tab.
+
+.  Monitor the *installationStatus* field in the status.
++
+If the value of *Completed nodes* matches the number of labeled nodes, then the installation is completed. The status also contains a list of nodes where the installation is completed.
```

Comment:
same comment as for line 74 of modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc